### PR TITLE
An 997/anchor collateral wh adds

### PR DESCRIPTION
**Pull request by amasucci13**
- Added new wormhole transactions and bATOM provide/withdraws to collateral model
- Fixed whsAVAX decimal issue in oracle_prices
- Fixed dupe bug in liquidations

### CODE REVIEW

File: models/terra/gold/terra__oracle_prices.sql [hunk 1]
```diff
@@ -185,8 +185,12 @@ SELECT
   ee.block_timestamp,
   ee.currency,
   l.address_name AS symbol,
-  pp.price / ee.price_usd AS luna_exchange_rate,
-  ee.price_usd,
+  CASE WHEN ee.currency = 'terra1z3e2e4jpk4n0xzzwlkgcfvc95pc5ldq0xcny58'
+  THEN pp.price / (ee.price_usd * POW(10,2))
```
Comment (forgxyz) [inline models/terra/gold/terra__oracle_prices.sql:189]: I anticipate the on-chain oracle will fix the 6 vs 8 decimal issue at some point soon. Should we blanket adjust by 2, or add conditional for the time being?
I.e. we know AVAX is highly unlikely to crash to under $2, so could change this to only adjust by 10^2 when the price feed < 2.

This way, the table won't need a full-refresh when (if) the Price Oracle corrects.

Comment (forgxyz) [reply models/terra/gold/terra__oracle_prices.sql:189]: The $2 logic applies to `price_usd` and I now see this comment is on `luna_exchange_rate`. Same idea applies.

**Review by forgxyz — APPROVED** [review]
A couple thoughts on dealing with wshAVAX using 8 instead of 6, commented in-line. 
Otherwise, all build with a `full-refresh` and passed tests.
LGTM!

File: models/terra/anchor/anchor__collateral.sql [hunk 2]
```diff
@@ -106,7 +110,107 @@ AND block_timestamp :: DATE >= (
...
+SELECT 
+a.blockchain,
+a.block_id,
+a.block_timestamp,
+a.chain_id,
+a.tx_id,
+a.msg_index,
+'provide' AS event_type,
+action_log :borrower::STRING AS sender,
+CASE WHEN event_attributes :collaterals [0] :denom::STRING = 'terra1z3e2e4jpk4n0xzzwlkgcfvc95pc5ldq0xcny58' --whsAVAX
```
[the remark this answers is not in the document]
Comment (amasucci13) [reply models/terra/anchor/anchor__collateral.sql:144]: I think we should hardcode for now. Decimals will be in available in new contract_metadata table. When that's released, we can get the decimals from there.

File: models/terra/gold/terra__oracle_prices.sql (hunk 1 above)
Comment (amasucci13) [reply models/terra/gold/terra__oracle_prices.sql:189]: We'll have token decimals in the new table, so if decimals = 8 then adjust 10^2, and we can remove the hardcoding.

**Pull request merged**